process_scenario_weo_2024.R

```diff
@@ -87,11 +76,21 @@ weo_2024_fig_chptr_3_raw <-
   )
 
 iea_global_ev_2024_raw <-
-  readr::read_csv(
-    file = iea_global_ev_2024_raw_full_filepath,
-    show_col_types = FALSE
+  read_xlsx(
```

Comment:
prefer explicit name spacing, e.g.

```suggestion
  readxl::read_xlsx(
```